jso/apis/src/main/java/org/teavm/jso/util/function/JSConsumer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright 2023 Bernd Busse.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.util.function;
+
+import java.util.Objects;
+
+@FunctionalInterface
+public interface JSConsumer<T> {
```

Comment:
I guess `@JSFunctor` annotation required here

Reply:
Yes, this annotation was required.
Somehow it wasn't properly working without this interface extending `JSObject` as well. Without it, passing lambda functions were rejected as "incompatible type".